grants/casper_stats/m2.md

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
The contents of M2, M3 and M4 may need to be updated. They are similar to M1 contents, including the Milestone name/number.